coverage/.last_run.json

```diff
@@ -1,5 +1,5 @@
 {
   "result": {
-    "covered_percent": 72.64
+    "covered_percent": 80.1
```

Comment:
Unrelated to this pull request, but it's usually a good idea to add the coverage directory to your project gitignore file since they are generated files and make commits noisy.